scripts/screener/screener.runner.ts

```diff
@@ -86,32 +78,9 @@ export async function screenerRunner(screenerConfig: ScreenerRunnerConfig) {
       : undefined,
   });
 
-  await notifyIntegration({
-    commit,
-    url: checkUrl.url,
-    status: 'in_progress',
-    project: screenerConfig.projectRepo,
-    branch: branchName,
-  });
-}
-
-export async function cancelScreenerRun(
-  screenerConfig: ScreenerRunnerConfig,
-  conclusion: ScreenerProxyPayload['conclusion'] = 'cancelled',
-) {
-  // https://github.com/microsoft/azure-pipelines-tasks/issues/9801
-  const commit = process.env.SYSTEM_PULLREQUEST_SOURCECOMMITID;
-  // https://github.com/screener-io/screener-runner/blob/2a8291fb1b0219c96c8428ea6644678b0763a1a1/src/ci.js#L101
-  let branchName = process.env.SYSTEM_PULLREQUEST_SOURCEBRANCH || process.env.BUILD_SOURCEBRANCHNAME;
-
-  await notifyIntegration({
-    commit,
-    url: 'https://screener.io/',
-    status: 'completed',
-    project: screenerConfig.projectRepo,
-    conclusion,
-    branch: branchName,
-  });
-
-  console.log(`cancelled screener run ${screenerConfig.projectRepo}`);
+  if (screenerRun.conclusion === 'skipped') {
```

Comment:
It's preferable to use early exit which lowers the complexity of the code, something like following:

```ts
if (screenerRun.conclusion === 'skipped') {
    console.log('Screener test skipped.');
    return;
  }
  if (screenerRun.conclusion === 'in_progress') {
    console.log('Screener test in progress.');
    return;
  }
```